### PR TITLE
doc(release): prepare release notes for MAP 22.10.4

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release notes for Centreon MAP 22.10.4 release.

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [x] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
 
+### 22.10.4
+
+Release date : `soon`
+
+#### Bug fixes
+
+- Fixed an issue preventing to create MAP with API
+- Fixed an issue that caused server to fail to start when trying to load empty output.
+- Fixed links when bendpoints are empty cause map to become not editable.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:28]:
```suggestion
- Fixed issue where empty endpoints caused maps to become not editable.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -17,6 +17,17 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.4
+
+Release date : `soon`
+
+#### Bug fixes
+
+- Fixed an issue preventing to create MAP with API
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:26]:
```suggestion
- Fixed an issue preventing users to create a map with the API
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -17,6 +17,17 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Fixed an issue that caused server to fail to start when trying to load empty output.
+- Fixed links when bendpoints are empty cause map to become not editable.
+- Fixed display label and use ressource name properties for links.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:29]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -17,6 +17,17 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+### 22.10.4
+
+Release date : `soon`
+
+#### Bug fixes
+
+- Fixed an issue preventing users to create a map with the API
+- Fixed an issue that caused server to fail to start when trying to load empty output.
+- Fixed links when bendpoints are empty cause map to become not editable.
+- Fixed display label and use ressource name properties for links.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:29]:
```suggestion
- Resource name properties are now used for links when saving maps.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -17,6 +17,17 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.4
+
+Release date : `soon`
+
+#### Bug fixes
+
+- Fixed an issue preventing users to create a map with the API
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:26]:
```suggestion
- Fixed an issue preventing users to create a map with the API.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -18,6 +18,17 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon MAP
 
+### 22.10.4
+
+Release date : `soon`
+
+#### Bug fixes
+
+- Fixed an issue preventing users to create a map with the API
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:27]:
```suggestion
- Fixed an issue preventing users to create a map with the API.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by smau-centreon — DISMISSED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]

**Pull request merged**